docs/index.rst

```diff
@@ -19,52 +19,48 @@ user-defined types among other features.
 With Solidity you can create contracts for uses such as voting, crowdfunding, blind auctions,
 and multi-signature wallets.
 
-.. note::
-    The best way to try out Solidity right now is using
-    `Remix <https://remix.ethereum.org/>`_
-    (it can take a while to load, please be patient). Remix is a web browser
-    based IDE that allows you to write Solidity smart contracts, then deploy
-    and run the smart contracts.
+Language Documentation
+----------------------
+
+If you are new to the concept of smart contracts we recommend you start with :ref:`an example smart contract <simple-smart-contract>` written
+in Solidity, followed by :ref:`the basics of blockchains <blockchain-basics>`
+and details of the :ref:`Ethereum Virtual Machine <the-ethereum-virtual-machine>`.
+
+When you are ready for more detail, we recommend you read the :doc:`"Solidity in Depth" <solidity-in-depth>` and :doc:`"Solidity by Example" <solidity-by-example>` sections to learn the core concepts of the language.
```

Comment:
Are you sure "solidity by example" should be mentioned that late? Most people probably don't even read the evm details or blockchain basics.

Reply:
OK fair point @chriseth try these changes.